fluent-langneg/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## Unreleased
+
+  - …
+
+## fluent-langneg 0.0.1
+
+  - (05d2487c) fluent-langneg 0.0.1
```

Comment:
This is my bad: I shouldn't have included the SHAs in other changelogs.  I've since stopped doing it.  Changelogs should describe changes between named versions and they don't have to correspond to commits. Let's remove the SHA of the commit here and squash everything into a single commit.